From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Fri, 15 Apr 2016 10:44:11 +0200
Subject: virt-sandbox: document how to pass arguments to commands

The example was already correct but the synopsis didn't mention '--' is
required.
---
 bin/virt-sandbox.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/bin/virt-sandbox.c b/bin/virt-sandbox.c
index 9495e85..d6a441c 100644
--- a/bin/virt-sandbox.c
+++ b/bin/virt-sandbox.c
@@ -345,7 +345,9 @@ virt-sandbox - Run cmd under a virtual machine sandbox
 
 =head1 SYNOPSIS
 
-virt-sandbox [OPTIONS...] COMMAND [CMDARG1 [CMDARG2 [...]]]
+virt-sandbox [OPTIONS...] COMMAND
+
+virt-sandbox [OPTIONS...] -- COMMAND [CMDARG1 [CMDARG2 [...]]]
 
 =head1 DESCRIPTION
 
